### DISCUSSION TITLE
PlaceholderAPI Bungeecord

### DISCUSSION THREAD

**Discussion by Boryte** [PlaceholderAPI/PlaceholderAPI#744] [(3) Ideas]
### Confirmation

- [X] I checked the Issues and Pull request tab for any existing issues or PRs.
- [X] My Feature Request is for **PlaceholderAPI** and not any expansion or other plugin.

### Type

Major Code improvement (Will affect Servers)

### Description

So at this moment PlaceholderAPI can only be used and installed on a single spigot server. My feature request expands that.
Currently, if you have a big network and rely on external premium spigot plugins you are very limited what you can do even with an API. My suggestion is that you are able to parse placeholders through entire bungeecord. So installing placeholderapi on bungeecord (like on other backends) will allow you to parse a minigame statistic for example to be used and seen on the hub server without using extremely complicated methods or using an entire plugin just for the sake of placeholders.
This would dramatically improve entire plugin and put it on another level compared to other competition since there's no similar system available on the market.

Example:
I have a custom Survival Games plugin that relies on PlaceholderAPI for use of placeholders. It's in bungee mode. It means every game is hosted on a different server. I want my players to share coins from the game and check each other statistics from Survival Games even while in the hub or in the other games. This would be possible with PlaceholderAPI bungee and parsing placeholders across the network to different servers,

**Comment by Andre601** [comment]
This is already planned for v3.
See #552 for current plans.
